lib/editor/result.coffee

```diff
@@ -3,18 +3,31 @@
 
 module.exports =
 class Result
+  constructor: (@editor, range, opts={}) ->
+    if not opts.type then opts.type = 'inline'
+    @type = opts.type
+    @disposables = new CompositeDisposable
+    @createView opts
+    @initMarker range
+    @text = @getText()
+    @disposables.add @editor.onDidChange (e) => @validateText e
+
   fadeIn: ->
     @view.classList.add 'ink-hide'
     @timeout 20, =>
       @view.classList.remove 'ink-hide'
 
-  inlineView: ({error, content, fade}) ->
+  createView: ({error, content, fade}) ->
     @view = document.createElement 'div'
-    @view.classList.add 'ink', 'inline', 'result'
+    @view.classList.add 'ink', 'result'
+    switch @type
+      when 'inline'
+        @view.classList.add 'inline'
+        @view.style.position = 'absolute'
+        @view.style.top = -@editor.getLineHeightInPixels() + 'px'
+        @view.style.left = '10px'
```

Comment:
Yeah it'd just be the other two. It might not even be worthwhile tbh, I'll leave it up to you